### PR TITLE
source controller

### PR DESCRIPTION
**Pull request by adrienjt**
This PR is a minor UX improvement that mirrors the target controller. Also, the getting started guide now uses namespaced source/targets.

ClusterSources and Sources CRDs allow the user to easily add cluster-scoped and namespaced sources. They replace the use of klum in the getting started guide and end-to-end test. Service accounts, cluster role bindings and role bindings can still be configured directly instead. The source controller can be disabled (see Helm flag value and hacky feature gate) if not compatible with the user's security policy.

Signed-off-by: adrienjt <adrienjt@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,11 +88,11 @@ done
 
 At this point, multicluster-scheduler is installed in both clusters, but neither is a source or target yet.
 
-1. Target clusters need service accounts and RBAC rules to autenticate and authorize source clusters, cf. [Creating Service Accounts and RBAC Rules in Target Clusters](#1-creating-service-accounts-and-rbac-rules-in-target-clusters), below.
+1. Target clusters need service accounts and RBAC rules to autenticate and authorize source clusters. The source controller can configure those for you, cf. [Creating ClusterSources and Sources in Target Clusters](#1-creating-clustersources-and-sources-in-target-clusters), below.
```
Comment (sfiligoi) [inline README.md:91]: "The source controller can ..." is confusing; seems to imply that you do this in the source cluster (while in reality it is done in the target cluster).

I would recommend expanding it to "The source controller in  the target cluster can ..."
I know it is a bit more verbose (and repetitive), but would be much easier to understand for a novice user, in my opinion.

File: README.md [hunk 2]
```diff
@@ -246,50 +269,51 @@ At this point, cluster1 has an identity in cluster2, credentials to be authentic
...
+- A namespaced Target references a kubeconfig secret in the same namespace. Multicluster-scheduler will use the kubeconfig to interact with resources in the target cluster in that namespace (except to view the cluster-scoped ClusterSummary, as explained above), so it must be authorized to do so there (i.e., cluster2 must have a Source for cluster1's identity in that namespace, or a RoleBinding between the `multicluster-scheduler-source` ClusterRole and cluster1's identity).
 
-ClusterTargets and Targets can also target their local cluster, e.g., for cloud bursting. In that case, they don't reference a kubeconfig secret, but specify `spec.self=true` instead. In our case, apply the following two ClusterTargets in cluster1:
+ClusterTargets and Targets can also target their local cluster, e.g., for cloud bursting. In that case, they don't reference a kubeconfig secret, but specify `spec.self=true` instead. In our case, apply the following two Targets in cluster1:
```
Comment (sfiligoi) [inline README.md:276]: Can we keep the ClusterTarget example?
It is still supported, right?

If you want to discourage that deployment model, maybe put it as an optional section?

File: README.md [hunk 3]
```diff
@@ -246,50 +269,51 @@ At this point, cluster1 has an identity in cluster2, credentials to be authentic
...
 
-- A namespaced Target associates a name with a reference to a kubeconfig secret (name only, in the same namespace). Multicluster-scheduler will use the kubeconfig to interact with resources in the target cluster in the eponymous namespace (except to view the cluster-scoped ClusterSummary, as explained above), so it must be authorized to do so there (i.e., there must be a RoleBinding between the `multicluster-scheduler-source` ClusterRole and cluster1's identity in that namespace).
+- A namespaced Target references a kubeconfig secret in the same namespace. Multicluster-scheduler will use the kubeconfig to interact with resources in the target cluster in that namespace (except to view the cluster-scoped ClusterSummary, as explained above), so it must be authorized to do so there (i.e., cluster2 must have a Source for cluster1's identity in that namespace, or a RoleBinding between the `multicluster-scheduler-source` ClusterRole and cluster1's identity).
```
Comment (sfiligoi) [inline README.md:274]: It is not clear if one can mix ClusterTarget's and Target's.
I would assume it is possible, at least for distinct  target clusters, but would be good to have that explicitly stated.

**Comment by sfiligoi** [comment]
The documentation looks mostly good, although I would like to see my comments above addressed.
No opinion on the code changes.

File: README.md (hunk 2 above)
Comment (adrienjt) [reply README.md:276]: ClusterTarget is supported. Following the principle of least privilege, I switched to namespaced Targets for the getting started guide, but you're right, I should include a ClusterTarget example in a collapsible section, to mirror the ClusterSource example above.

File: README.md (hunk 3 above)
Comment (adrienjt) [reply README.md:274]: :+1:

File: README.md (hunk 1 above)
Comment (adrienjt) [reply README.md:91]: :+1:

**Pull request merged**